scripts/weave.yaml

```diff
@@ -8,10 +8,9 @@ items:
       annotations:
         cloud.weave.works/launcher-info: |-
           {
-            "server-version": "master-12a0a52",
             "original-request": {
-              "url": "/k8s/v1.7/net.yaml?k8s-version=Q2xpZW50IFZlcnNpb246IHZlcnNpb24uSW5mb3tNYWpvcjoiMSIsIE1pbm9yOiI5IiwgR2l0VmVyc2lvbjoidjEuOS4xIiwgR2l0Q29tbWl0OiIzYTFjOTQ0OWE5NTZiNjAyNmYwNzVmYTMxMzRmZjkyZjdkNTVmODEyIiwgR2l0VHJlZVN0YXRlOiJjbGVhbiIsIEJ1aWxkRGF0ZToiMjAxOC0wMS0wNFQxMTo1MjoyM1oiLCBHb1ZlcnNpb246ImdvMS45LjIiLCBDb21waWxlcjoiZ2MiLCBQbGF0Zm9ybToibGludXgvYW1kNjQifQpTZXJ2ZXIgVmVyc2lvbjogdmVyc2lvbi5JbmZve01ham9yOiIxIiwgTWlub3I6IjkiLCBHaXRWZXJzaW9uOiJ2MS45LjEiLCBHaXRDb21taXQ6IjNhMWM5NDQ5YTk1NmI2MDI2ZjA3NWZhMzEzNGZmOTJmN2Q1NWY4MTIiLCBHaXRUcmVlU3RhdGU6ImNsZWFuIiwgQnVpbGREYXRlOiIyMDE4LTAxLTA0VDExOjQwOjA2WiIsIEdvVmVyc2lvbjoiZ28xLjkuMiIsIENvbXBpbGVyOiJnYyIsIFBsYXRmb3JtOiJsaW51eC9hbWQ2NCJ9Cg==",
-              "date": "Tue Jan 16 2018 15:44:10 GMT+0000 (UTC)"
+              "url": "/k8s/v1.8/net.yaml?k8s-version=1.9.2",
```

Comment:
Did you run conformance on the weave CNI setup?

Reply:
Working on that now.

Reply:
Weave conformance looks good: https://scanner.heptio.com/4756ad7eae2837ea42d18945ee8ba962/diagnostics/

I hit an issue with the calico conformance run here: https://scanner.heptio.com/b595ccdd7364b98fb3e47994c69ed8f9/diagnostics/ working on troubleshooting

Reply:
Does this occur on an older version of calico?

Reply:
A second run was clean: https://scanner.heptio.com/8b884a35b8460437dd1cf87448288b8a/diagnostics/

I'm going to re-spin a new cluster and test again.